### PR TITLE
Add L 5.4 support

### PR DESCRIPTION
**Pull request by MCMatters**
Please review

### CODE REVIEW

**Comment by ramondj** [comment]
+1

**Comment by MathSmath** [comment]
+1

**Comment by manticorp** [comment]
+1

**Comment by manticorp** [comment]
Please update this asap! It's the last package in my stack stopping me from updating to 5.4!

File: src/Mariuzzo/LaravelJsLocalization/LaravelJsLocalizationServiceProvider.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public function boot()
     public function register()
     {
         // Bind the Laravel JS Localization command into the app IOC.
-        $this->app['localization.js'] = $this->app->share(function ($app) {
+        $this->app->singleton('localization.js', function ($app) {
```
Comment (rmariuzzo) [inline src/Mariuzzo/LaravelJsLocalization/LaravelJsLocalizationServiceProvider.php]: @MCMatters  I'm checking if this is supported by Laravel 5.0 – 5.4

Comment (rmariuzzo) [inline src/Mariuzzo/LaravelJsLocalization/LaravelJsLocalizationServiceProvider.php]: @MCMatters just verified, and I can see it there: https://laravel.com/api/5.0/Illuminate/Contracts/Container/Container.html#method_singleton

**Comment by rmariuzzo** [comment]
I will create the test for Laravel 5.4 and see if it works properly. If all good, then I will merge into master.

**Pull request merged**